brave/browser/brave_content_browser_client.cc

```diff
@@ -316,10 +319,22 @@ void BraveContentBrowserClient::ExposeInterfacesToRenderer(
 #endif
 }
 
+void BraveContentBrowserClient::RegisterInProcessServices(
```

Comment:
@bridiver I have realised this was guarded with an OS_ANDROID define in chrome's content browser client.